recipes/sdl2_mixer/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "SDL2_mixer" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "5a24f62a610249d744cbd8d28ee399d8905db7222bf3bdbc8a8b4a76e597695f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.libsdl.org/projects/SDL_mixer/release/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
```

Comment:
I left these out at first, but then AppVeyor tried to build sdl2_mixer two times. Once as sdl2_mixer-2.0.1-vc14-1 and once as sdl2_mixer-2.0.1-1 (thus without a vc flag). I don't know how to solve this other than to explicitly add these lines?